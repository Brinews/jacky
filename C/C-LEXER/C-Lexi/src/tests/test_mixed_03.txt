
??? END_OF_FILE MINUS PLUS NUM ID

1 x08
1x 08
1x0 8
1x09

3BCx16
3BC x16
3BCx 16
3BCx1 6
3BCx17
