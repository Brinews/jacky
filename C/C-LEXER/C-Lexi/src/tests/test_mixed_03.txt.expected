{ , ERROR , 2}
{ , ERROR , 2}
{ , ERROR , 2}
{END , ID , 2}
{ , ERROR , 2}
{OF , ID , 2}
{ , ERROR , 2}
{FILE , ID , 2}
{MINUS , ID , 2}
{PLUS , ID , 2}
{NUM , ID , 2}
{ID , ID , 2}
{1 , NUM , 4}
{x08 , ID , 4}
{1 , NUM , 5}
{x , ID , 5}
{0 , NUM , 5}
{8 , NUM , 5}
{1 , NUM , 6}
{x0 , ID , 6}
{8 , NUM , 6}
{1 , NUM , 7}
{x09 , ID , 7}
{3BCx16 , BASE16NUM , 9}
{3 , NUM , 10}
{BC , ID , 10}
{x16 , ID , 10}
{3 , NUM , 11}
{BCx , ID , 11}
{16 , NUM , 11}
{3 , NUM , 12}
{BCx1 , ID , 12}
{6 , NUM , 12}
{3 , NUM , 13}
{BCx17 , ID , 13}
{ , END_OF_FILE , 15}
